website/blog/index.html

```diff
@@ -15,13 +15,21 @@
 -->
 
 <!-- TODO: Avoid boilerplate. -->
+<div class="blog-listing">
```

Comment:
Since the paper isn't exactly a blog, maybe we could write a very short blog post describing that the paper was accepted and link to it from there?

Reply:
Good idea. Done. Can you take a look?

Reply:
Looks great! I fixed a small bug in the line height of the note. This is my first time pushing a change to someone else's PR, so hopefully it doesn't break everything :crossed_fingers: